pkg/operator/ceph/object/user.go

```diff
@@ -99,7 +99,7 @@ func GetUser(c *Context, id string) (*ObjectUser, int, error) {
 }
 
 // CreateUser creates a new user with the information given.
-func CreateUser(c *Context, user ObjectUser) (*ObjectUser, int, error) {
+func CreateUser(c *Context, user ObjectUser, IsSystemUser bool) (*ObjectUser, int, error) {
```

Comment:
`CreateUser`, as a public api for this package, should have unit tests.

Reply:
There is already unit test for create user api. Do you mean to check this functionality as well?